### DISCUSSION TITLE
0.9.4 Snapshot

### DISCUSSION THREAD

**Discussion by mariuz** [mariuz/flamerobin#285] [General]
## What's Changed
* Merge Dev into master  by @mariuz in https://github.com/mariuz/flamerobin/pull/264
* Possibility to shut down the database and start it up again. #211 by @Jdochoa in https://github.com/mariuz/flamerobin/pull/270
* Add support for update or insert by @Jdochoa in https://github.com/mariuz/flamerobin/pull/271
* Merge Dev into master by @mariuz in https://github.com/mariuz/flamerobin/pull/272
* Optimization by @Jdochoa in https://github.com/mariuz/flamerobin/pull/273
* Deterministic clause of stored functions not reflected in generated D… by @Jdochoa in https://github.com/mariuz/flamerobin/pull/274
* Flamerobin hang on executing comment on #266 by @Jdochoa in https://github.com/mariuz/flamerobin/pull/275
* Update master from Dev by @mariuz in https://github.com/mariuz/flamerobin/pull/276
* small typo in databasehandler.cpp by @real-dam in https://github.com/mariuz/flamerobin/pull/278
* Int128 fixes on 32-bit arm and other non-mainstream architectures by @real-dam in https://github.com/mariuz/flamerobin/pull/279
* Update the backup/restore options  #212  by @Jdochoa in https://github.com/mariuz/flamerobin/pull/282
* Backup/Restore improvement by @Jdochoa in https://github.com/mariuz/flamerobin/pull/283

**Full Changelog**: https://github.com/mariuz/flamerobin/compare/0.9.3.12...0.9.4

<hr /><em>This discussion was created from the release <a href='https://github.com/mariuz/flamerobin/releases/tag/0.9.4'>0.9.4 Snapshot</a>.</em>

**Comment by luronumen** [comment]
Hi @mariuz 

Could you generate the build of this version for Windows 64 bits?

Thanks in advance,
Luciano

**Comment by PizzaProgram** [comment]
For 32 bit too please !
